src/ILCompiler/src/Program.cs

```diff
@@ -12,6 +12,7 @@
 using Internal.TypeSystem.Ecma;
 
 using Internal.CommandLine;
+using System.IO;
```

Comment:
Nit: Move this up above with the other `using System.*` usings. We organize system usings together first in a source file, ideally sorted alphabetically.

Reply:
Where do we need this `using`?